lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -190,6 +190,9 @@ def call(params)
             create_non_existing_user_groups: create_non_existing_user_groups,
             user_groups_cache: nil
           }
+          GoodData.gd_logger.info("Synchronizing in mode=#{mode}, data_rows=#{new_users.size}")
```

Comment:
Do we need project_id or process_id or execution_id to know this log belong to which execution?

Reply:
No, We don't need add them because the log auto add execution_id when logging splunk. We can use k8s container id or execution_id to know it belong to which execution:
Jan  2 08:07:44 fe0f6b6bb23cbc09839040d67b71ca8cfcebee38f678ffd900000019-z5xqd lcm_ruby_brick: I, [2020-01-02T08:07:44.614764 #7]  INFO -- Synchronizing in mode=add_to_organization, data_rows=2:  api_version=2.1.8 log_v=0 component=lcm.ruby status=action_in_progress execution_id=5e0d96901be1fc1eb7d4c1c1 brick=users action=GoodData::LCM2::SynchronizeUsers time=990.084